.pre-commit-config.yaml

```diff
@@ -2,22 +2,22 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.2.0
     hooks:
-      # - id: trailing-whitespace
-      # - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
       - id: check-yaml
       - id: debug-statements
-      # - id: double-quote-string-fixer
       # - id: name-tests-test
+      #   args: ["--django"]
```

Comment:
if you're using pytest to run these I would not recommend the `--django` setting -- it allows files to sneak through without being run

Reply:
I left it commented because I need to investigate what the complains are. I may need to rename things around.

Reply:
the default or `--pytest-test-first` are what I'd recommend :+1: